packages/es-dev-server/src/create-middlewares.ts

```diff
@@ -1,28 +1,29 @@
-import koaStatic from 'koa-static';
-import koaEtag from 'koa-etag';
-import koaCompress from 'koa-compress';
 import chokidar from 'chokidar';
```

Comment:
What happened to this file? Alphabetically reordered?

Reply:
Yep, organizeImports VS Code setting. I had it turned on when working on this PR, one day I might be able to make this a build-thing where the CI will include organizing imports in the formatting, but for now it is sadly dependent on .. the user's editor :<